configuration/pih/scripts/global/zl.js

```diff
@@ -349,4 +349,48 @@ function setRequiredForTextInputs(requiredWidgetIds) {
 */
 function setRequiredForRadioButton(radioButtonId, value) {
   jq(radioButtonId).find("input[type=radio]").prop('required', value);
+}
+
+
+
+// Function to check if any checkboxes are selected
+function anyCheckboxesSelected(containerSelector) {
+  return jq(`${containerSelector} input[type="checkbox"]:checked`).length > 0;
+}
+
+/**
+ * Updates the required attribute of the last checkbox in the given container.
```

Comment:
Reading the code, I think this code makes the latest checkbox in container required if any of the earlier checkboxes in the container has been checked?

Makes sense to me if that's the desired behavior, though would be good to that to this description to make it clear.